src/Http/Controllers/Corporation/MarketController.php

```diff
@@ -25,7 +25,7 @@
 use Seat\Services\Repositories\Corporation\Market;
```

Comment:
# Corporation Market
- sorting by total is not working properly (is there a missing attribute ?)
- sorting by type (sell/buy) is throwing an exception
```
DataTables warning: table id=corporation-market - Exception Message:

		SQLSTATE[42S22]: Column not found: 1054 Unknown column 'bid' in 'order clause' (SQL: select 

                --

                -- Select All

                --

                *,



                --

                -- Start stationName Lookup

                --

                CASE

                when a.location_id BETWEEN 66015148 AND 66015151 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id-6000000)

                when a.location_id BETWEEN 66000000 AND 66014933 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id-6000001)

                when a.location_id BETWEEN 66014934 AND 67999999 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id-6000000)

                when a.location_id BETWEEN 60014861 AND 60014928 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id)

                when a.location_id BETWEEN 60000000 AND 61000000 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id)

                when a.location_id >= 61000000 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id)

                else (SELECT m.itemName FROM mapDenormalize AS m

                    WHERE m.itemID = a.location_id) end

                    AS stationName from corporation_orders as a inner join `invTypes` on `a`.`type_id` = `invTypes`.`typeID` inner join `invGroups` on `invTypes`.`groupID` = `invGroups`.`groupID` where `a`.`corporation_id` = 98413060 order by `bid` asc limit 10 offset 0)
```
- making a research is throwing an exception - both are probably related
```
DataTables warning: table id=corporation-market - Exception Message:

			SQLSTATE[42S22]: Column not found: 1054 Unknown column 'bid' in 'where clause' (SQL: select count(*) as aggregate from (select '1' as `row_count` from corporation_orders as a inner join `invTypes` on `a`.`type_id` = `invTypes`.`typeID` inner join `invGroups` on `invTypes`.`groupID` = `invGroups`.`groupID` where `a`.`corporation_id` = 98413060 and (LOWER(`issued`) LIKE %t% or LOWER(`bid`) LIKE %t% or LOWER(`volEntered`) LIKE %t% or LOWER(`price`) LIKE %t% or LOWER(`price`) LIKE %t% or LOWER(`typeName`) LIKE %t%)) count_row_table)
```